serving/docs/lmi_new/deployment_guide/README.md

```diff
@@ -57,3 +57,19 @@ These libraries expose features and capabilities through different APIs and mech
 With LMI's built-in handlers, there is no need to learn each library and write custom code to leverage the features and optimizations they offer.
 We expose a unified configuration format that allows you to easily switch between libraries as they evolve and improve over time.
 As the ecosystem grows and new libraries become available, LMI can integrate them and offer the same consistent experience.
+
+## Feature Matrix
+
+|                                       | HuggingFace Accelerate | LMI_dist (8.0.0) | DeepSpeed (0.12.6) | TensorRTLLM (0.7.1)    | TransformersNeuronX (2.16.0) | vLLM (0.2.7) |
+| ------------------------------------- | ---------------------- | ---------------- | ------------------ | ---------------------- | ---------------------------- | ------------ |
+| DLC                                   | DeepSpeed              | DeepSpeed        | DeepSpeed          | LMI TRTLLM             | LMI Neuron                   | DeepSpeed    |
+| current handler                       | huggingface            | huggingface      | deepspeed          | tensorrtllm            | transformersneuronx          | huggingface  |
+| support quantization                  | BitsandBytes/GPTQ      | GPTQ/AWQ         | SmoothQuant INT8   | SmoothQuant, AWQ, GPTQ | INT8                         | GPTQ/AWQ     |
+| GPU machine supported                 | G4/G5/P4D/P5           | G5/P4D/P5        | G5/P4D/P5          | G5/P4D/P5              | INF2/TRN1                    | G4/G5/P4D/P5 |
```

Comment:
nit: Should this just be `Instance type`? (inf2 is not a gpu?)

Reply:
addressed